recommender/recommender.py

```diff
@@ -976,16 +975,16 @@ def add_xml_to_node(self, node):
         """
         node.tag = 'recommender'
 
-        node.set('intro_enabled', str(self.intro_enabled))
-        node.set('DISABLE_DEV_UX', str(self.client_side_settings['DISABLE_DEV_UX']))
-        node.set('ENTRIES_PER_PAGE', str(self.client_side_settings['ENTRIES_PER_PAGE']))
-        node.set('PAGE_SPAN', str(self.client_side_settings['PAGE_SPAN']))
+        node.set('intro_enabled', 'true' if (self.intro_enabled) else 'false')
+        node.set('disable_dev_ux', 'true' if (self.client_configuration['disable_dev_ux']) else 'false')
+        node.set('entries_per_page', str(self.client_configuration['entries_per_page']))
+        node.set('page_span', str(self.client_configuration['page_span']))
```

Comment:
Huh. I just tried it myself, and you're right. That's surprises me, but I guess the only thing you can do is to convert them to strings like you're doing here.